File: 02. C++/09. Funciones/Funciones.cpp
#include <iostream>
#include <string>

using namespace std;

/* Función */
void funcion1() {
    cout << "Hola Mundo" << endl;
}

/* Función con atributos */
void funcion2(string atributo1, string atributo2) {
    cout << atributo1+" "+atributo2 << endl;
}

/* Función con retorno */
string funcion3() {
    return "Hola Mundo";
}

/* Función con atributos y retorno*/
string funcion4(string atributo1, string atributo2) {
    return atributo1 + " " + atributo2;
}

int main() {

    /* Función 1 */
    funcion1();

    /* Función 2 */
    funcion2("Hola","Mundo");

    /* Función 3 */
    cout << funcion3() << endl;

    /* Función 4 */
    cout << funcion4("Hola","Mundo") << endl;

}

File: 05. Lenguajes no clasificados/02. Visual Basic/09. Funciones/Funciones.vb
Module Module1

    ' Función 
    Sub funcion1()

        Console.WriteLine("Hola Mundo")

    End Sub

    ' Función con atributos
    Sub funcion2(atributo1 As String, atributo2 As String)

        Console.WriteLine(atributo1 + " " + atributo2)

    End Sub

    ' Función con retorno
    Function funcion3()

        Return "Hola Mundo"

    End Function

    ' Función con atributos y retorno
    Function funcion4(atributo1 As String, atributo2 As String)

        Return atributo1 + " " + atributo2

    End Function

    Sub Main()

        ' Función 1 
        funcion1()

        ' Función 2 
        funcion2("Hola", "Mundo")

        ' Función 3
        Console.WriteLine(funcion3())

        ' Función 4
        Console.WriteLine(funcion4("Hola", "Mundo"))

        Console.In.ReadLine()

    End Sub

End Module
